firebase_admin/credentials.py

```diff
@@ -95,44 +120,54 @@ def __init__(self, cert):
         try:
             self._g_credential = service_account.Credentials.from_service_account_info(
                 json_data, scopes=_scopes)
+            self._g_credential_async = service_account_async.Credentials.from_service_account_info(
+                json_data, scopes=_scopes)
         except ValueError as error:
             raise ValueError('Failed to initialize a certificate credential. '
                              'Caused by: "{0}"'.format(error))
 
     @property
-    def project_id(self):
+    def project_id(self) -> str:
         return self._g_credential.project_id
 
     @property
-    def signer(self):
+    def signer(self) -> google.auth.crypt.Signer:
         return self._g_credential.signer
 
     @property
-    def service_account_email(self):
+    def service_account_email(self) -> str:
         return self._g_credential.service_account_email
 
-    def get_credential(self):
-        """Returns the underlying Google credential.
+    def get_credential(self) -> service_account.Credentials:
+        """Returns the underlying Google synchronous credential.
 
         Returns:
-          google.auth.credentials.Credentials: A Google Auth credential instance."""
+          google.auth.credentials.Credentials: A Google Auth synchronous credential instance."""
         return self._g_credential
 
+    def get_credential_async(self) -> service_account_async.Credentials:
+        """Returns the underlying Google asynchronous credential.
+
+        Returns:
+          google.auth._credentials_async.Credentials: A Google Auth asynchronous credential
+          instance."""
+        return self._g_credential_async
 
 class ApplicationDefault(Base):
     """A Google Application Default credential."""
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Creates an instance that will use Application Default credentials.
 
-        The credentials will be lazily initialized when get_credential() or
-        project_id() is called. See those methods for possible errors raised.
+        The credentials will be lazily initialized when get_credential(), get_credential_async()
```

Comment:
Suggest code font for these literal references to functions, like "..when <code>get_credential()</code>,..."

Reply:
@jkyle109 I think you can try double ` here. See https://github.com/firebase/firebase-admin-python/blob/master/firebase_admin/messaging.py#L130

Reply:
Sounds good, Done!